### DISCUSSION TITLE
Jokes app - Slower than expected

### DISCUSSION THREAD

**Discussion by ashvin777** [remix-run/remix#961] [Show and tell / tips]
NOTE - If https://remix-jokes.lol/jokes is not up to date, then this can be ignored.

I was trying out the https://remix-jokes.lol/jokes app to test the overall performance and noticed that sub-components are loading very slow. Are we not prefetching the sub-routes components? 

Look at the below GIF where I am clicking on the "Add" button and I see the form after some time. The wait time is more than what I was expecting.

![Dec-09-2021 14-47-17](https://user-images.githubusercontent.com/8143154/145369172-d6b95818-969e-4ecc-81d6-7ed29d00ab3e.gif)

**Comment by MichaelDeBoey** [comment]
I've created a PR to fix this:
- In the tutorial: https://github.com/remix-run/remix/pull/967
- In the live app: https://github.com/remix-run/remix-jokes/pull/11
